### DISCUSSION TITLE
error "Offset is outside the bounds of the DataView" help

### DISCUSSION THREAD

**Discussion by CorruptionVR** [yume-chan/ya-webadb#394] [General]
I connected to a 4.0 android tablet on the web demo, and now it is giving me this error on both tablets. 4.0, and 11.0. how can I fix this?

**Comment by yume-chan** [comment]
Please file your issue using the [template](https://github.com/yume-chan/ya-webadb/issues/new?assignees=&labels=bug%2Cdemo&template=demo_bug.yml).
